guest/api/framework.go

```diff
@@ -1,5 +1,16 @@
 package api
 
+type QueueingHint int
```

Comment:
This refers to https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/framework/types.go#L256-L263. Where should this be placed?

Reply:
Why do we need this in the first place? We don't support QueueingHint in the extension yet.